ostree/ostree_noselinux.go

```diff
@@ -0,0 +1,14 @@
+// +build !containers_image_ostree_stub,mac_stub
+
+package ostree
+
+import "os"
+
+type mandatoryAccessControl interface {
+	Close()
+	ChangeLabels(root string, fullpath string, fileMode os.FileMode) error
+}
```

Comment:
Please move this type definition into a single location not gated by a build tag (`ostree_dest.go` would work), to ensure that the two definitions in `ostree_noselinux.go` and `ostree_selinux.go` can’t get out of sync.

And use private method names (lower-case `close` and `changeLabels`).